ios/Classes/MapboxMapController.swift

```diff
@@ -97,10 +97,10 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
                 result(nil)
                 //Style could happen to been ready before the map was ready due to the order of methods invoked
                 //We should invoke onStyleLoaded
-                if isStyleReady && !isStyleReadyNotified {
+                if isStyleReady && !onStyleLoadedCalled {
                     if let channel = channel {
+                        onStyleLoadedCalled = true
```

Comment:
Shouldn't we first invoke the callback and only the set the boolean?

Reply:
changed that on purpose. As soon as this if is entered the callback will be sent here. So this could avoid calling onStyleLoaded more than once.